compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1475,9 +1475,6 @@ class OMR_EXTENSIBLE CodeGenerator
    bool getSupportsArraySet() {return _flags1.testAny(SupportsArraySet);}
    void setSupportsArraySet() {_flags1.set(SupportsArraySet);}
 
-   bool getSupportsArraySetToZero() {return _flags3.testAny(SupportsArraySetToZero);}
```

Comment:
My reasoning was exactly that - a codegen may only want to support setting to zero rather than arbitrary setting and having this degree of freedom seems useful in the abstract and since it is already implemented I don't think we need to throw it away.